binutils-2.39/ld/ldmain.c

```diff
@@ -52,6 +52,7 @@
 
 #define GITOID_LENGTH_SHA1 20
 #define GITOID_LENGTH_SHA256 32
+#define MAX_LONG_SIZE_STRING_LENGTH 256
```

Comment:
Would something like MAX_FILE_SIZE_STRING_LENTH be better, since you are looking to specify the size of the file in string (not the size of long)?